contrib/kube-local/kube-local

```diff
@@ -0,0 +1,1592 @@
+        # will return 1 from git command and will fail
+        ${cmd} &> /dev/null
+    else
+        # DO not use exec_cmd here. If branch already exists like master
+        # will return 1 from git command and will fail
+        ${cmd}
+    fi
+
+    return 1
+
+}
+
+function install_project() {
+    ###############################################################
+    # Description:                                                #
+    #   Build and install a project into golang src dir           #
+    #                                                             #
+    #   PARAMS:                                                   #
+    #       ${1} - project name                                   #
+    #           example: kubernetes                               #
+    #                                                             #
+    #       ${2} - project golang dir                             #
+    #           example: ${HOME}/go/src/github/kubernetes         #
+    #                                                             #
+    #       ${3} - project golang src dir                         #
+    #           example: github                                   #
+    #                                                             #
+    #       ${4} - github branch name                             #
+    #           example: master                                   #
+    #                                                             #
+    #       ${5} - github url for the project                     #
+    #           example: https://github.com/kubernetes/kubernetes #
+    #                                                             #
+    #       ${6} - action - possible values:                      #
+    #           MAKE_ONLY                                         #
+    #             - execute: make                                 #
+    #                                                             #
+    #           MAKE_AND_MAKE_INSTALL                             #
+    #             - execute: make && make install                 #
+    #                                                             #
+    #   Just another example:                         Args number #
+    #   -----------------------                      ------------ #
+    #        install_project \                                    #
+    #            ${RUNTIME_NAME} \                   # ${1}       #
+    #            ${RUNTIME_GOLANG_DIR} \             # ${2}       #
+    #            ${RUNTIME_GOLANG_SRC_GITHUB} \      # ${3}       #
+    #            ${RUNTIME_VERSION} \                # ${4}       #
+    #            ${RUNTIME_GITHUB_URL}               # ${5}       #
+    #            "MAKE_AND_MAKE_INSTALL"             # ${6}       #
+    ###############################################################
+
+    ############ Some local vars ##################################
+    SUPPORTED_ACTIONS="MAKE_ONLY, MAKE_AND_MAKE_INSTALL"
```

Comment:
nit: I find the second `MAKE` redundant. I'd suggest:
`SUPPORTED_ACTIONS="MAKE_ONLY, MAKE_AND_INSTALL"`